tests/main/ubuntu-core-device-reg/task.yaml

```diff
@@ -31,6 +31,9 @@ execute: |
         ubuntu-core-18-64)
             snap known serial | MATCH "model: ubuntu-core-18-amd64"
             ;;
+        ubuntu-core-18-arm-*)
+            snap known serial | MATCH "model: ubuntu-core-18-$gadget_name"
```

Comment:
not for this PR, but maybe we could use the newly minted `snap model` command for this kind of thing instead, I happen to know it's of _excellent_ quality :grinning: